platform/pom.xml

```diff
@@ -29,17 +29,11 @@
     <packaging>pom</packaging>
 
     <properties>
-        <cxf.version>3.0.4</cxf.version>
         <cxf.jaxb.version>2.2.6</cxf.jaxb.version>
-        <commons-io.version>2.1</commons-io.version>
         <joda-time.version>2.2</joda-time.version>
         <joda-convert.version>1.7</joda-convert.version>
-        <org.slf4j.version>1.7.1</org.slf4j.version>
         <osgi.version>5.0.0</osgi.version>
```

Comment:
I can probably do that, but I worry it might be confusing. Intellij isn't
smart enough to follow that property to its definition in the imported pom.

If you think that's okay, I'll make that change.

On Thu, Aug 6, 2015 at 1:26 PM Phil Klinefelter <notifications@github.com>
wrote:

> In platform/pom.xml
> <https://github.com/codice/ddf/pull/91#discussion_r36461059>:
>
> >          <joda-time.version>2.2</joda-time.version>
> >          <joda-convert.version>1.7</joda-convert.version>
> > -        <org.slf4j.version>1.7.1</org.slf4j.version>
> >          <osgi.version>5.0.0</osgi.version>
>
> The osgi version property is define by the karaf parent pom which is
> imported. We should be able to remove this property.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codice/ddf/pull/91/files#r36461059>.
>